metric_tank/metric_tank.go

```diff
@@ -228,6 +228,13 @@ func main() {
 		log.Fatal(4, "failed to connect to NSQLookupds. %s", err)
 	}
 
+	InitCluster(*instance, !*dryRun, metrics, stats)
+
+	err = InitCassandra()
```

Comment:
any particular reason why the init cassandra code was moved down? just curious. it seems to not depend on the result of InitCluster().